docs/src/changelog.md

```diff
@@ -14,9 +14,9 @@ If your currently installed *Quality-time* version is not the latest version, pl
 
 ## [Unreleased]
 
-### Fixed
+### Added
 
-- Todo
+- Allow for measuring the time since the last analysis date of a Bill-of-Materials (BOM) in Dependency-Track using the 'source up-to-dateness' metric. This can be configured using the new 'event type' parameter. Possible values are 'last BOM upload', 'last BOM analysis', or 'all types' (oldest of last BOM analysis and last BOM upload). Closes [#9764](https://github.com/ICTU/quality-time/issues/9764).
```

Comment:
```suggestion
- Allow for measuring the time since the last analysis date of a Bill-of-Materials (BOM) in Dependency-Track using the new 'event type' parameter of the 'source up-to-dateness' metric. Closes [#9764](https://github.com/ICTU/quality-time/issues/9764).
```